blackberggroup/VA-DOE
# Feat/rl 7895552096/dta updates

## What type of PR is this? (check all applicable)


Additional edit requests to DTA page

## Related Tickets & Documents
- Monday Task : https://blackberg-group.monday.com/boards/1931719392/pulses/7895552096

## QA Instructions, Screenshots, Recordings

Please review the attached Word doc linked in the Monday ticket. Verify the edits on this list compared to our dev sandbox. 
There is one edit request that is not updated intentionally due to an issue with design cohesiveness: "Update banner by enlarging the logo and placing it in the middle. Delete all text on the right"

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _site/assets/img/accelerator/dta-mission2.jpg
 - assets/img/accelerator/dta-mission2.jpg
 - _includes/talent-accelerator/impact.html
 - _includes/talent-accelerator/mission.html
 - _includes/talent-accelerator/recruitment.html
 - _includes/talent-accelerator/timeline.html
 - _site/assets/css/styles.css
 - _site/assets/js/mustache-fellows-render.js
 - _site/assets/sass/vha/pages/talent-accelerator/recruitment/_index.scss
 - _site/diffusion-talent-accelerator/index.html
 - assets/css/styles.css
 - assets/js/mustache-fellows-render.js
 - assets/sass/vha/pages/talent-accelerator/recruitment/_index.scss
